server/src/main/java/CSC4410/CovidTracker/controller/NameSearchController.java

```diff
@@ -0,0 +1,31 @@
+package CSC4410.CovidTracker.controller;
+
+import CSC4410.CovidTracker.model.County;
+import CSC4410.CovidTracker.operation.query.CountyLocationQuery;
+import CSC4410.CovidTracker.operation.query.CountySearchQuery;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+@RestController
+public class NameSearchController {
+
+    private final CountySearchQuery query = new CountySearchQuery(null);
+
+    @RequestMapping(value = "/search")
```

Comment:
i think renaming the URL like /location/search to be  consistent with the previous /location url